pkg/mkverification-raw-efi/verify

```diff
@@ -399,6 +399,14 @@ tpm2_pcrread >> "$REPORT/summary.log"
 
 cp "/root/etc/eve-release" "$REPORT"
 
+find /sys/kernel/iommu_groups/ -type l > "$REPORT/iommu_groups.out"
+
+watchdogs=$(find /dev -name "watchdog*" | grep -vw "/dev/watchdog")
+if [ "$(echo \"${watchdogs}\" | wc -l)" -ne 1 ]; then
```

Comment:
Note that there can be zero /dev/watchdog - you report that as multiple!
Probably want to put how many in the message.
If so doing 
count=$(echo "$watchdogs" | wc -l)
seems to also avoid the shellcheck complaint.